sdk/servicebus/azure-servicebus/migration_guide.md

```diff
@@ -1,164 +1,408 @@
+choice and start sending/receiving messages.
+
+#### Approachability
+By having a single entry point, the `ServiceBusClient` helps with the discoverability of the API
+as you can explore all available features through methods from a single client, as opposed to searching
+through documentation or exploring namespace for the types that you can instantiate.
+Whether sending or receiving, using sessions or not, you will start your applications by constructing the same client.
+
+#### Consistency
+We now have methods with similar names, signature and location to create senders and receivers.
+This provides consistency and predictability on the various features of the library.
 
-This document is intended for users that are familiar with v0.50 of the Python SDK for Service Bus library (`azure-servicebus 0.50.x`) and wish 
-to migrate their application to v7 of the same library.
-
-For users new to the Python SDK for Service Bus, please see the [readme file for the azure-servicebus](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/README.md).
-
-## General changes
-Version 7 of the azure-servicebus package is the result of our efforts to create a client library that is user-friendly and idiomatic to the Python ecosystem.
-Alongside an API redesign driven by the new [Azure SDK Design Guidelines for Python](https://azure.github.io/azure-sdk/python_introduction.html#design-principles), 
-the latest version improves on several areas from v0.50.
+### Client constructors
 
-Note: The large version gap is in order to normalize service bus SDK versions across our languages, as they consolidate on structure as well.
+- While we continue to support connection strings when constructing a client, the main difference is when using Azure Active Directory.
+We now use the new [`azure-identity`](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/README.md) library
+to share a single authentication solution between clients of different Azure services.
 
-### Specific clients for sending and receiving
-In v7 we've simplified the API surface, making two distinct clients, rather than one for each of queue, topic, and subscription:
-* `ServiceBusSender` for sending messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.html#azure.servicebus.ServiceBusSender)
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusSender)
-* `ServiceBusReceiver` for receiving messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.html#azure.servicebus.ServiceBusReceiver)
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusReceiver)
+In V0.50:
```

Comment:
Not blocking this PR, but thoughts for a future PR

We want to show 3 different things here
- That constructing the client using connection string has not changed
- Constructing client using SAS key and name is an upcoming feature
- Using AAD is a brand new feature

Consider treating these separately. For example:

> Constructing the client using the connection string is same in both versions.
`servicebus_client = ServiceBusClient.from_connection_string(conn_str)`

> Additionally, you can now use Azure Active Directory for authentication via the `azure-identity` library
`show code for AAD here`

> The option to construct the client using SAS key name and value is not available in the new version, but will be added in the near future.

Reply:
I love the way you organize the concepts, I have copied your words and tweaked a bit.